tests/bluetooth/gatt/src/main.c

```diff
@@ -89,8 +89,14 @@ static struct bt_gatt_attr test1_attrs[] = {
 
 static struct bt_gatt_service test1_svc = BT_GATT_SERVICE(test1_attrs);
 
-void test_gatt_register(void)
+ZTEST_SUITE(test_gatt, NULL, NULL, NULL, NULL, NULL);
+
+ZTEST(test_gatt, test_gatt_register)
 {
+	/* Ensure our test services are not already registered */
+	bt_gatt_service_unregister(&test_svc);
```

Comment:
@tristan-google @yperess why these new lines?

Reply:
because testcases with new API need to be independent and should not depent on previous tests, i.e. we should be ableto run the tests in random.